### ISSUE TITLE
[v2.7.0] User cannot reset password through reset password form

### ISSUE THREAD

**Issue by jm18457** [reactioncommerce/reaction#5725]
## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
User should be enter email and receive email with further instructions on how to change password.
User in turn receives 401 error, because of token middleware.
![image](https://user-images.githubusercontent.com/20977577/67005993-3c83ef80-f0e4-11e9-91f4-0267676a161c.png)

## Steps to Reproduce
Please provide starting context, i.e. logged in as a user, configure a particular payment method.
1. Have latest reaction.
2. Go to localhost:3000 and click forgot password.
3. Enter email. 
4. You should receive email, but you there will be 401 error.

Issue is caused by [token middleware](https://github.com/reactioncommerce/reaction/blob/master/imports/node-app/core-services/account/util/tokenMiddleware.js#L39), because it passes `"null"` token into [getUserFromMeteorToken](https://github.com/reactioncommerce/reaction/blob/master/imports/node-app/core-services/account/util/getUserFromMeteorToken.js#L32).

This particular mutation should be exempt from token middleware, because user is not logged in while asking for password reset email. 

## Possible Solution
Exclude  mutation `sendResetPasswordEmail` from token middleware.

### Versions
2.7.0

**Comment by jm18457** [comment]
I did some debugging and arrived to the following conclusions:
1. Issue is present with v2.7.0 and not v2.6.0.
2. Issue happens because route is not found in [login component](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/accounts/client/components/login.js#L24) `currentroute.name` is 'not-found'

login.js was not changed in update from 2.6 to 2.7, but I saw that register.js was changed and `no-meteor/register.js` was moved.
